config/application.rb

```diff
@@ -41,5 +41,7 @@ class Application < Rails::Application
 
     # Set a css_compressor so sassc-rails does not overwrite the compressor
     config.assets.css_compressor = nil
+
+    config.action_controller.include_all_helpers = false
```

Comment:
Why are we doing this now?

Reply:
This was because I needed the helpers to be coupled to the controllers (so I could have two `eligibility_answers` in different helpers depending on context). This may be moot now though if we decide to move the logic to the views